src/application/admin/service_test.go

```
@@ -0,0 +1,50 @@
+	"github.com/stretchr/testify/mock"
+)
+
+func TestService_GenerateToken(t *testing.T) {
+	mockJwt := new(mockjwt.JwtService)
+	expireTimeSec := 3600
+	svc := NewService(mockJwt, expireTimeSec)
+
+	t.Run("success", func(t *testing.T) {
+		userId := "user-123"
+		userType := "USER"
+		mockJwt.On("GenerateToken", mock.MatchedBy(func(token *entity.JwtToken) bool {
```

golangci-lint (1.64.8)

only cuddled expressions if assigning variable or using from line above

(wsl)